content/docs/quickstart/symphony.md

```diff
@@ -1,14 +1,18 @@
 ---
-title: Run a Symphony app
+title: Run a Symfony app
 ---
 
-Symfony is a framework for building web applications in PHP. Symfony uses Doctrine library to access database. Using Neon from Symfony + Doctrine is straightforward and differs nothing from using a vanilla postgresql.
+### Introduction
 
-First, obtain secret token from “Connection details” panel:
+Symfony is a framework for building web applications in PHP. Symfony uses Doctrine library to access database. Using Neon from Symfony + Doctrine is straightforward and differs nothing from using a vanilla PostgreSQL.
 
-1. Select Project you wish to use in the UI. Click on it.
-2. Click “Generate token” link.
-3. Substitue token placeholder below with this token.
+## Step 1 — Obtain Secret token from "Connection Details" Panel
+
+## Step 2 — Select Project in the UI
+
+## Step 3 — Click "Generate token" link
```

Comment:
But replace this one too

Reply:
@ololobus 

Replace with what information?

Reply:
With something more up to date like this https://github.com/neondatabase/website/pull/59#discussion_r894740431

Reply:
BTW, do you have an access to https://console.neon.tech ? I think it will be more easy to edit docs if you'll have a look on UI

Reply:
@alexandriastech , @ololobus is referring to the fact that there're no tokens anymore, and we're using proper password authentication, user can reset their password.
sorry for coming up with this so late.

Reply:
@ololobus @alexandriastech Yes, no issue, Anna! I am not familiar with Symfony and do have access to the Neon Console. 

I need clarification to make sure the steps are correct. 

So, are the steps changing overall to something like this,

## Step 1 — Set up a Neon Project

## Step 2 — Select Project in the UI

## Step 3 — Copy the connection string (`DATABASE_URL`)  

and so on,

**or are they just changing ## Step 3??**

## Step 1 — Obtain Secret token from "Connection Details" Panel

## Step 2 — Select Project in the UI

## Step 3 — Copy the connection string (`DATABASE_URL`) 

## Step 4 — Substitute token placeholder below with this token

Reply:
the first set of steps provided by you, @alexandriastech , looks good (1. set up project, 2. select project, 3. copy the conn string.)

NB, password is shown only once, when the password is generated, because Neon stores the encrypted passwords. Neon shows the password right after a user creates the project, and also when the user clicks 'reset password' button.

Reply:
@stepashka @ololobus I updated the steps https://github.com/neondatabase/website/pull/59/commits/54f447c5eeb0a7a46baabd20509c647eb03a381a

Did you want to add the explanation, "password is shown only once, when the password is generated, because Neon stores the encrypted passwords. Neon shows the password right after a user creates the project, and also when the user clicks 'reset password' button" to symphony.md?

If so, is this an explanation that needs to go across the rest of your documentation?

Reply:
> If so, is this an explanation that needs to go across the rest of your documentation?

I think that we can create a separate docs section like 'Users and passwords management' and refer this section from other places. And put all these details there. So in these guides we can say just 'Get password and copy connection string from the dashboard (for details refer to this docs section)' and don't duplicate the same info

But his could be a separate PR

Reply:
@ololobus Ok, that makes sense, a separate PR is a good idea.